ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf

```diff
@@ -103,15 +104,58 @@ If.dmic {
 		Type String
 		Empty "${var:DeviceDmic}"
 	}
-	False.If.Dmic0 {
-		Condition {
-			Type ControlExists
-			Control "name='Dmic0 Capture Volume'"
+	False {
+		If.Dmic0 {
+			Condition {
+				Type ControlExists
+				Control "name='Dmic0 Capture Volume'"
+			}
+			True.BootSequence [
+				cset "name='Dmic0 Capture Volume' 70%"
+				cset "name='Dmic0 Capture Switch' on"
+			]
+		}
+		If.Dmic0Drc {
```

Comment:
Could we move DRC/Beamformer Ifs to dsp.conf to have all related things in one place? Define a macro.

Reply:
Yep, it makes sense. I will start also with SoundWire codec PC's SOF processing setup, so I made for the macros the endpoint name a parameter.